example/web/main.dart

```diff
@@ -0,0 +1,3 @@
+void main() {
+  print('hello world!     dadasd');
```

Comment:
nit: can probably leave this as just `hello world!` ;)

Reply:
Oh, I updatet it to see the changes and forget to change it back. :-)